### PR TITLE
disable 'Update' button on submit

### PR DESCRIPTION
**Pull request by nickpell**
Prevent the user from making double-submissions (which in turn creates duplicate subscriptions)

@gcapalbo @proteusvacuum 

@esoergel I wish I had attended the IoC AotW before implementing this feature ... I probably wouldn't refactor it until it starts to become unwieldy, but I think the code would have come out better if written more linearly.

### CODE REVIEW

File: corehq/apps/domain/forms.py [hunk 1]
```diff
@@ -1290,13 +1290,17 @@ def __init__(self, domain, web_user, *args, **kwargs):
...
-                crispy.Submit(
-                    self.slug,
-                    ugettext_noop('Update')
+        return (
+            crispy.Hidden('slug', self.slug),
+            FormActions(
+                crispy.ButtonHolder(
+                    crispy.Submit(
+                        self.slug,
+                        ugettext_noop('Update'),
```
Comment (proteusvacuum) [inline corehq/apps/domain/forms.py]: I think this can be a regular old `ugettext`. I don't think this will be translated in this case.

**Comment by proteusvacuum** [comment]
:+1: Made one comment, but it wasn't something you changed (seems like we need to look at a lot of the `ugettext_noop`s in the accounting code).

**Comment by nickpell** [comment]
:white_check_mark:

File: corehq/apps/domain/forms.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/domain/forms.py]: I think it's worth handling all these updates in a separate PR

Comment (proteusvacuum) [inline corehq/apps/domain/forms.py]:
```py
>>> ugettext("moi aussi")
me too
```

**Comment by esoergel** [comment]
:+1:

**Pull request merged**